### PR TITLE
Adding enigma2 media player

### PR DESCRIPTION
**Pull request by fbradyirl**
**Description:**
Adding enigma2 docs. (this is based on comments from a stale review here: #5984 ).

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21271

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by fbradyirl** [comment]
apologies @klaasnicolaas I had to clean up my fork so ended up opening this fresh PR.

File: source/_components/media_player.enigma2.markdown
Comment (klaasnicolaas) [inline source/_components/media_player.enigma2.markdown:47]: It's missing the default name: `Enigma2 Media Player`

File: source/_components/media_player.enigma2.markdown
Comment (klaasnicolaas) [inline source/_components/media_player.enigma2.markdown]: You can remove this line with `port`

File: source/_components/media_player.enigma2.markdown
Comment (klaasnicolaas) [inline source/_components/media_player.enigma2.markdown]: You can remove this line with `name`

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/media_player.enigma2.markdown
Comment (fbradyirl) [reply source/_components/media_player.enigma2.markdown]: Done.

File: source/_components/media_player.enigma2.markdown
Comment (fbradyirl) [reply source/_components/media_player.enigma2.markdown]: Done.

File: source/_components/media_player.enigma2.markdown
Comment (fbradyirl) [reply source/_components/media_player.enigma2.markdown:47]: Done.

**Comment by fbradyirl** [comment]
@klaasnicolaas all comments addressed (I think).

**Review by klaasnicolaas — DISMISSED** [review]
✅ approved
⚠️ `ha_release` must be updated before merging

File: source/_components/media_player.enigma2.markdown
Comment (rytilahti) [inline source/_components/media_player.enigma2.markdown]: Worth noting about the autodiscovery here, too?

File: source/_components/media_player.enigma2.markdown
Comment (rytilahti) [inline source/_components/media_player.enigma2.markdown:19]: Are there limitations which devices it can be installed? Worth mentioning here? Is the setup so straightforward, that there is no need to describe how to set it up here?

**Review by rytilahti — DISMISSED** [review]
Looks good to me, added a note about a couple of potential improvements!

File: source/_components/media_player.enigma2.markdown
Comment (fbradyirl) [reply source/_components/media_player.enigma2.markdown]: Good point, yes I will add that.

File: source/_components/media_player.enigma2.markdown
Comment (fbradyirl) [reply source/_components/media_player.enigma2.markdown:19]: No, there are no limitations. In fact, you will struggle to buy an Enigma2 based box in the past 15 years which does not come pre-installed with OpenWebif. There is literally no setup involved for the end user. It is enabled on port 80 by default with no auth.

**Review by rytilahti — APPROVED** [review]
Can be merged when the parent gets merged, thanks!

**Pull request merged**
